### PR TITLE
Increase build natives granularity and install deps from requirements file

### PR DESCRIPTION
**Pull request by 0lmi**

### CODE REVIEW

**Review by oleorhagen — APPROVED** [review]
Looks sane to me.

Mac test seems to be failing though

File: Makefile [hunk 1]
```diff
@@ -44,17 +44,25 @@ PLATFORMS := darwin linux windows
 
 GO_LDFLAGS_WIN = -ldflags "-X github.com/mendersoftware/mender-artifact/cli.Version=$(VERSION) -linkmode=internal -s -w -extldflags '-static' -extld=x86_64-w64-mingw32-gcc"
 
-build-natives:
+build-native-linux:
+	 @arch="amd64";
+	 @echo "building linux";
+	 @env GOOS=linux GOARCH=$$arch \
+        $(GO) build -a $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME)-linux ;
```
Comment (lluiscampos) [inline Makefile:51]: There is a mix of spaces and tabs (it works fine because the line with spaces is preceded by `\`)-

**Review by lluiscampos — APPROVED** [review]
Good idea. Thanks

> Mac test seems to be failing though

@oleorhagen The failures are unrelated. Failing [in master](https://gitlab.com/Northern.tech/Mender/mender-artifact/-/jobs/3222115375) as we speak.

**Comment by 0lmi** [comment]
> Mac test seems to be failing though

Looks like a glitch. Job restart makes it passing 🙃

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

**Comment by 0lmi** [comment]
Due to we are still waiting for the CI pushed another change to make it even nicer 🙃

**Pull request merged**